resolwe_bio/processes/workflows/cut_and_run_beta.py

```diff
@@ -0,0 +1,460 @@
+                    "the wrong mate begins upstream. Consider "
+                    "that to be concordant. Default: True.",
+                )
+                rep_se = BooleanField(
+                    label="Report single ended",
+                    default=False,
+                    description="If paired alignment can not be found Bowtie2 tries "
+                    "to find alignments for the individual mates. Default: False.",
+                )
+                minins = IntegerField(
+                    label="Minimal distance",
+                    default=200,
+                    description="The minimum fragment length (--minins) for valid paired-end alignments. "
+                    "Default: 10.",
+                )
+                maxins = IntegerField(
+                    label="Maximal distance",
+                    default=700,
+                    description="The maximum fragment length (--maxins) for valid paired-end alignments. "
+                    "Default: 700.",
+                )
+                discordantly = BooleanField(
+                    label="Report discordantly matched read",
+                    default=False,
+                    description="If both mates have unique alignments, but the alignments do "
+                    "not match paired-end expectations (orientation and relative distance) then "
+                    "alignment will be reported. Useful for detecting structural variations. "
+                    "Default: False.",
+                )
+
+            class OutputOptions:
+                """Output options."""
+
+                no_unal = BooleanField(
+                    label="Suppress SAM records for unaligned reads",
+                    default=True,
+                    description="When true, suppress SAM records for unaligned reads. Defaults to true.",
+                )
+
+            pe_options = GroupField(PEOptions, label="Paired-end options")
+            output_options = GroupField(OutputOptions, label="Output options")
+
+        class NormalizationOptions:
+            """Spike-in normalization options."""
+
+            skip_norm = BooleanField(
+                label="Skip normalization",
+                default=False,
+                description="Skip the spike-in normalization step of BigWig output. "
+                "Use this if you don't provide a spike-in. Defaults to false.",
+            )
+            scale = FloatField(
+                label="Scale factor",
+                default=10000,
+                description="Magnitude of the scale factor. The scaling factor is calculated by "
+                "dividing the scale with the number of features in BEDPE (scale/(number of features)). "
+                "Default: 10000.",
+                disabled="normalization_options.skip_norm == True",
+            )
+
+        class SubsampleOptions:
+            """Subsampling options."""
+
+            n_reads = IntegerField(
```

Comment:
I think we should make downsampling optional - please add additional checkbox option for that. The default should be True.